### PR TITLE
Add velero namespace config variable

### PR DESCRIPTION
**Pull request by twiest**
- [x] Add velero namespace config variable
- [x] Add unit tests
- [x] Manually test new option

### CODE REVIEW

File: pkg/test/manager/fakemanager.go [hunk 1]
```diff
@@ -0,0 +1,102 @@
+package manager
```
Comment (csrwng) [inline pkg/test/manager/fakemanager.go:1]: Could this have been generated by gomock?

**Comment by twiest** [comment]
@csrwng I believe I've addressed your comments. PTAL.

File: pkg/controller/velerobackup/velerobackup_controller.go [hunk 2]
```diff
@@ -223,14 +237,18 @@ func (r *ReconcileBackup) createVeleroBackupObject(namespace string, t metav1.Ti
...
+			Namespace: r.veleroNamespace,
 		},
 		Spec: velerov1.BackupSpec{
+			VolumeSnapshotLocations: []string{},
 			IncludedNamespaces: []string{
 				namespace,
 			},
 			ExcludedResources: defaultExcludedBackupResources,
 		},
+		Status: velerov1.BackupStatus{
```
Comment (staebler) [inline pkg/controller/velerobackup/velerobackup_controller.go:249]: What is the purpose of including the `Status`? For most resource types, the create of the resource will clear out the status.

Comment (twiest) [reply pkg/controller/velerobackup/velerobackup_controller.go:249]: Yeah, this was something new. So apparently the most recent version of Velero has a validation hook that errors without this. I discovered this during my manual tests and adding this line made it work.

**Pull request merged**